index.html

```diff
@@ -23,47 +23,66 @@
             <div class="inner">
                 <h1>Gci16.fossasia.org</h1>
                 <h2>GCI Website 2016</h2>
-                <a href="https://github.com/fossasia/gci16.fossasia.org" class="button"><small>View project on</small> GitHub</a>
+                <a href="//github.com/fossasia/gci16.fossasia.org" class="button">
+                    <small>View project on</small> GitHub</a>
             </div>
         </header>
 
         <section class="main-content">
             <div class="container">
-            		<section id="contributors">
-			            <h2>CONTRIBUTORS</h2>
-			            <div id="contributors-list">
-            			</div>
-        				</section>
+                <section id="contributors">
+                    <h2>CONTRIBUTORS</h2>
+                    <div id="contributors-list">
+                    </div>
+                </section>
                 <div class="students-section">
                     <h1 class="head"><span>Students</span></h1>
                     <hr>
                     <div class="row">
-                        <div class="col-lg-3 col-md-3 col-sm-4">
+                        <div class="col-sm-4 col-md-3">
```

Comment:
However this exists in main.css, and it will be lost if `col-lg-3` is removed.

```css
.row .col-lg-3 {
	width: 33.33%;
	float: left;
	padding-left: 30px;
	padding-right: 30px;
	padding-bottom: 15px;
}
```

I think the css `.row .col-lg-3` could be changed to `.row .col-lg-3, .row col-md-3` to work around that, so the rule doesnt need to be disabled ..?